data/registry/instrumentation-go-goyek.yml

```diff
@@ -0,0 +1,19 @@
+# cSpell:ignore goyek, otelgoyek, Pająk
+title: goyek instrumentation
+registryType: instrumentation
+language: go
+tags:
+  - go
+  - instrumentation
+  - goyek
+license: MIT
+description: Package otelgoyek provides OpenTelemetry instrumentation for goyek.
+authors:
+  - name: Robert Pająk
+    url: https://github.com/pellared
+urls:
+  repo: https://github.com/goyek/x/tree/main/otelgoyek
+  docs: https://pkg.go.dev/github.com/goyek/x/otelgoyek
+createdAt: 2025-02-26
+isNative: false
+isFirstParty: true
```

Comment:
I don't think that this should be marked as first-party. I've created an issue to track the discussion and possible followup fixed:

- #6428

Reply:
I did this in accordance with 

https://github.com/open-telemetry/opentelemetry.io/blob/61efc4685844e84b0b6995aff7e7eb0c27f3bd7f/templates/registry-entry.yml#L18